ms/common_setup.cmd

```diff
@@ -1,13 +1,19 @@
 @echo off
+setlocal EnableDelayedExpansion
 
+set CUSTOM_BUIL_DIR=%~1
+if "%CUSTOM_BUIL_DIR%"=="" (
```

Comment:
%VAR% - used when evaluation doesn't take place.
!VAR! -used when var value is calculated(related to the setlocal EnableDelayedExpansion)